backend/api/routers/alexa.ts

```diff
@@ -1,14 +1,13 @@
 import bodyParser from '@voiceflow/body-parser';
 import express from 'express';
 
-import { BODY_PARSER_SIZE_LIMIT } from '@/backend/constants';
 import { ControllerMap, MiddlewareMap } from '@/lib';
 
 export default (middlewares: MiddlewareMap, controllers: ControllerMap) => {
   const router = express.Router();
 
+  router.use(bodyParser.text({ type: '*/*' }));
   router.use(middlewares.alexa.verifier);
-  router.use(bodyParser.json({ limit: BODY_PARSER_SIZE_LIMIT }));
```

Comment:
the `middlewares.alexa.verifier` does a JSON Parse on the body already, no need for bodyParser

Reply:
might be worth adding a comment to the source